src/sconsx/tools/builddir.py

```diff
@@ -35,12 +35,16 @@ def __init__(self, config):
     def default(self):
         """Set the default directory values for build_prefix."""
         # self._default['build_prefix']= pj(self.config.dir[0], "build-" + platform.name)
-        if 'CONDA_BUILD' in os.environ:
-            self._conda_build = True
-            prefix = self._default['build_prefix'] = CONDA_PREFIX
-        else:
-            self._conda_build = False
-            prefix = self._default['build_prefix'] = pj(self.config.dir[0],"build-scons")
+        #if 'CONDA_BUILD' in os.environ:
+        #    self._conda_build = True
+        #    prefix = self._default['build_prefix'] = CONDA_LIBRARYPREFIX
+        #else:
+        #    self._conda_build = False
+        #    prefix = self._default['build_prefix'] = pj(self.config.dir[0],"build-scons")
+
+        prefix = self._default['build_prefix'] = pj(self.config.dir[0],"build-scons")
```

Comment:
Not sure I understand. On conda, we want to put the lib, include and bin directly in the PREFIX and LIB_PREFIX and not in build_scons, isn't it?

Reply:
No, see deploy. The cleanest solution is to build in build-scons and install on system. See install.py for thus